src/app/productConfig/productConfigModal.tpl.html

```diff
@@ -43,9 +43,11 @@ <h4 class="panel-title border-bottom-small" translate>
               <label class="btn btn-radio" ng-click="$ctrl.itemConfig.amount = 5000" ng-class="{'active': $ctrl.itemConfig.amount === 5000}"><span class="number">$5000</span></label>
               <label class="btn btn-default-form u-textLeft custom-amount">
                 <div class="form-group form-group-default u-inline">
-                  <input class="form-control form-control-subtle number" min="0" step="any" type="text" ng-model="$ctrl.itemConfig.amount">
+                  <input class="form-control form-control-subtle number" min="1" type="number" name="amount" ng-model="$ctrl.itemConfig.amount" required>
                 </div>
               </label>
+              <span class="help-block" validation="$ctrl.itemConfigForm.amount" show-error="required" translate>You must enter an amount</span>
+              <span class="help-block" validation="$ctrl.itemConfigForm.amount" show-error="min" translate>Amount must be at least $1</span>
```

Comment:
Are we ever going to accept non USD currencies? Not a blocker for this PR unless you can think of a generic error message.